src/renderer/components/FtContextMenu/FtContextMenu.css

```
@@ -158,13 +158,28 @@
 
+.submenu::before {
+  position: absolute;
+  inset-block: 0;
+  inset-inline-end: 100%;
+  inline-size: 48px;
+  clip-path: polygon(0 16px, 100% 0, 100% 100%);
+  content: '';
+}
+
 .submenusOpenStart .submenu,
 :global(.isLocaleRightToLeft) .submenu {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)